src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
+  let initialObj = { ...state };
+
+  for (const { type, extraData: toAdd, keysToRemove: toDelete } of actions) {
```

Comment:
make more descriptive namings toAdd -> propertiesToAdd

Reply:
> make more descriptive namings, toAdd --> propertiesToAdd

Thanks for your feedback;